packages/api/tests/models/test-rules-model.js

```diff
@@ -311,6 +362,7 @@ test.serial('create a kinesis type rule, using existing event source mappings',
 
   await rulesModel.delete(rule);
   await rulesModel.delete(newRule);
+  await deleteKinesisEventSourceMappings();
```

Comment:
I don't doubt that it's necessary, but why did you add these to most of the tests? To ensure test isolation?

Reply:
Yes, when I was writing the test the other way this was needed and I figured there's no reason to go delete it since it's actually a good change to isolate the tests and may help someone down the road.